.github/actions/deploy-macosx/action.yml

```diff
@@ -26,8 +26,7 @@ runs:
 
         brew uninstall --force --ignore-dependencies gcc gcc@7 gcc@8 gcc@9 gcc@10 gcc@11 gcc@12 gcc@13 gcc@14 little-cms2 maven openblas r
         brew install boost ccache swig autoconf-archive automake cmake libomp libtool libusb ant nasm xz pkg-config sdl2 gpg1 bison flex perl ragel binutils gradle gmp isl libmpc mpfr geoip pcre ssdeep yajl
-        brew link --force libomp
-
```

Comment:
Why do you need to remove this line?

Reply:
This line is harmful when compiling with gcc, because the linker will pick libomp (that is llvm implementation of openmp) instead of libgomp. In the case of Pytorch and other presets using llvm, it's ok but not necessary because cmake, at least in its recent versions, can find libomp in the keg path.
If you want to keep it, it'd be better in a  `else` block of a an `if [[ -n ${CI_DEPLOY_NEED_GCC:-} ]];` block

Reply:
Yes, let's do that